app/helpers/application_helper/toolbar/container_images_center.rb

```diff
@@ -35,6 +35,23 @@ class ApplicationHelper::Toolbar::ContainerImagesCenter < ApplicationHelper::Too
           :url_parms => "main_div",
           :enabled   => "false",
           :onwhen    => "1+"),
+        button(
```

Comment:
Yeah, can't refactor this here and still be able to tell what's being added for this feature.

Future refactor (and I'm sure it needs to be refactored for *all* the tool bar files)